### ISSUE TITLE
Question about DQN's loss

### ISSUE THREAD

**Issue by ku2482** [deepmind/bsuite#23]
Hi, I have one simple question about DQN's loss [here](https://github.com/deepmind/bsuite/blob/master/bsuite/baselines/tf/dqn/agent.py#L129).

Why do you use `tf.reduce_sum` instead of `tf.reduce_mean` here??
Are there some reasons for it? Have experiments in the paper been done calculating loss which sums over the batch??

Sorry for asking such a simple question, but I would really appreciate it if you answer my question.

Anyway, this is a great project !!
Thank you :)

**Comment by aslanides** [comment]
Hi, thanks for the question.

It is true that reduce_mean is more 'correct' in that it is invariant to the batch size. In terms of learning dynamics, I don't think it makes much difference since this loss is optimized on its own, not in combination with another (in contrast to say, actor-critic) In this case, a scaling by a constant factor will essentially get normalized away by an optimizer like Adam.

Having said that, I think reduce_mean is probably clearer/simpler/makes for easier comparisons, so I'll change that. Thanks!
